File: 2756-buy-two-chocolates/buy-two-chocolates.py
class Solution(object):
    def buyChoco(self, arr, key):
        ans = float('inf')

        for i in range(len(arr)):
            for j in range(len(arr)):
                if i != j:
                    total = arr[i] + arr[j]
                    ans = min(ans, total)

        fin = key - ans
        if(fin>=0):
            return fin
        else:
            return key
        

File: 2756-buy-two-chocolates/buy-two-chocolates.js
var buyChoco = function(arr, key) {
    let ans = Infinity;
        let fin;

        for (let i = 0; i < arr.length; ++i) {
            for (let j = 0; j < arr.length; ++j) {
                if (i !== j) {
                    const sum = arr[i] + arr[j];
                    ans = Math.min(ans, sum);
                }
            }
        }

        fin = key - ans;
        if(fin>=0){
        return fin;
        }
        else{
            return key;
        }
};